src/tracking_state.jl

```diff
@@ -51,6 +69,31 @@ function DownconvertedSignalCPU(num_ants::NumAnts{N}) where N
     )
 end
 
+"""
+$(SIGNATURES)
+
+DownconvertedSignalGPU that holds possible downconverted signal representations.
+The type is unknown, when TrackingState is initialized.
+It is determined based on the signal, which is given in the track
+function.
+"""
+struct DownconvertedSignalGPU{
```

Comment:
The DownconvertedSignalGPU is never used, isn't it? I guess we can get rid of this structure.